### DISCUSSION TITLE
What can cause symbols to fail to display?

### DISCUSSION THREAD

**Discussion by rcherny** [darlal/obsidian-switcher-plus#187] [Q&A] [answered]
So I have two vaults with virtually the same set of plugins/themes installed. On one, the command for "Open Symbols for selected suggestion or editor" successfully shows the Headers, indented, in a markdown document. In one, the command always displays an empty list with nothing showing.

Any thoughts on what may cause that?

**Comment by darlal** [comment]
An empty list is usually the result of the command acting upon a source file that doesn't not contain symbols, or, it cannot be parsed. For example, if you try to run that command when the active pane contains a PDF file, an empty list will be displayed.

A few things you can try:

1. What happens when you use the trigger character for `Open Symbols for selected suggestion or editor` instead of using the global command? To use the trigger character, open the switcher, search for a markdown file that you know has symbol content, use the up/down keys to highlight that file in the switcher, once it's highlighted, type the trigger character.
2. Make sure there aren't other plugins that's modifying the current focused pane
3. Check the developer console for any errors being reported. If you see any error please let me know.

**Reply by rcherny** [reply]
Thanks for the response. I'll take a look when I'm back from my holiday travels later this week. Thanks again,

**Reply by rcherny** [reply, marked as answer]
@darlal so I know it's been what, almost a year? lol — so this continued to plague me, but I finally had some time to debug. 

It's a little embarrassing, but I wanted to just close the loop: I had a facet "filter" set in the modal that excluded everything. 

I looked, and finally saw that "tags" was highlighted in the modal, and has been for a year! When I disabled that, the symbols appeared. So backstory: I have my Obsidian configured to move the tags to the frontmatter properties. I don't use inline tags very much. It seems that the Quick Switcher, when filtering by symbols such as tags (or if that's the app file cache, I don't know), doesn't find the tags in the frontmatter (bug? feature?). When I tested putting an inline tag in there, suddenly the symbols for the current editor showed when filtered by tags. 

Anyways, that doesn't bother me in the least — so I'm good now. Plus, I found that you can filter the symbols, which I didn't realize before 😅 But wow, sorry for not paying attention to the filter. Didn't even see it highlighted. May have been my theme 🤷‍♂️ 

Thanks again!

**Comment by darlal** [comment]
Well, I'm glad to hear that you got it figured out.. if a bit funny 😉 

As for the tags, they're retrieved via the Obsidian API so I would imagine it should contain tags from both sources for a given file, but I'll take a look at this.

I appreciate the update, thanks!
